mvp.feature

```diff
@@ -1102,3 +1104,39 @@ Feature: MVP features for NVA
       | Institution DNS               |
       | Administration ID             |
       | Feide Organization ID         |
+
+  @902
+  Scenario: User opens a Public Profile from a Public Page
+    Given the user has opened NVA
+    And sees a Public Page
+    When they click a contributor
+    Then the contributors public profile page is opened
+
+  @913
+  Scenario: User sees published publications
+    Given the user is logged in as Creator
+    And has opened My Publications page
+    When they click the navigation bar to see his published registrations
+    Then a list of all published registrations with the fields
+      | Title    |
+      | <Status> |
+      | Created  |
+    Examples:
+      | Status    |
+      | Deleted   |
+      | Published |
+    And each list item has a button Delete and Edit
+    And the edit button is enabled
+    And the Delete button is enabled if the registration is not marked as Deleted
+    And the navigation bar for unpublished registrations is enabled
+    And the navigation bar for published registrations is selected
+
+  @914
+  Scenario: User sees published publications
+    Given the user is logged in as Creator
+    And has opened the page My Publications
```

Comment:
Disse to linjene kan også byttes ut med denne:
`Given Creator opens My Publications`